### PR TITLE
check animations all stop bug.

### PR DESCRIPTION
**Pull request by wyklion**

### CODE REVIEW

**Comment by deltakosh** [comment]
can you provide a PG which higlights this issue? I see no problem in the current code

File: src/Animations/babylon.animatable.ts [hunk 1]
```diff
@@ -119,7 +119,7 @@
                         numberOfAnimationsStopped++;
                     }
 
-                    if (animations.length == numberOfAnimationsStopped) {
```
Comment (wyklion) [inline src/Animations/babylon.animatable.ts]: it has no means...
animations length x,  if numberOfAnimationsStopped = y, see (animations.splice(index, 1);) then animations.length = x-y;
the check animations.length == numberOfAnimationsStopped means (x-y == y).

**Comment by wyklion** [comment]
(animations.length == numberOfAnimationsStopped) it has no means...
animations length x, if numberOfAnimationsStopped = y, see (animations.splice(index, 1);) then animations.length = x-y;
the check animations.length == numberOfAnimationsStopped means (x-y == y).

File: src/Animations/babylon.animatable.ts (hunk 1 above)
Comment (wyklion) [inline src/Animations/babylon.animatable.ts]: the variable numberOfAnimationsStopped is useless..

Comment (deltakosh) [inline src/Animations/babylon.animatable.ts]: ok make sense :)

Comment (deltakosh) [inline src/Animations/babylon.animatable.ts]: Then numberOfAnimationsStopped has to be removed

**Comment by deltakosh** [comment]
Thanks!

**Pull request merged**